packages/api/src/module/write/email-sender/email-sending.write-module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import {SharedModule} from "@/write/shared/shared.module";
+import {SendEmailMessageCommandHandler} from "@/write/email-sender/application/send-email-message.command-handler";
+
+@Module({
+  imports: [SharedModule],
+  controllers: [],
+  providers: [SendEmailMessageCommandHandler]
```

Comment:
There is lack of EmailSender provider. 
You can also override it for mocked implementation in tests.